src/checks/consistency/single_greater_than_any.rs

```diff
@@ -0,0 +1,195 @@
+use crate::{ConsistencyCache, Flag, Timeseries, TimeseriesPair};
+
+/// Compares a single value to a higher resolution sequence, where the single value should never
+/// be greater than any value in the sequence (including an adjustment)
```

Comment:
```suggestion
/// Compares a single value to a sequence.
/// The single value should never be greater than any value in the sequence (including an adjustment)
```
Not sure what high resolution means in this context? Maybe this should be moved to the `cache` version?

Reply:
Higher time resolution, or simply that there are more points. The sentence does make sense without it though, so I'm happy to remove it though if you still feel it doesn't belong

Reply:
Yeah, I think it works better if this is mentioned in the cache check instead (as a potential use case for the check)?

Reply:
I don't understand what you mean, could you write a sample of what you would put in the doc for the cache version?

Reply:
You're right, having it on the cache version makes it more convoluted. I would still remove the `higher resoultion` bit, but add something like (in the `single_outside_sequence` doc)
```rust
/// One possible use case for this check is when `single` is an aggragated value,
/// for example, max hourly temperature compared against the minute observations for that hour.
```
What do you think?

Reply:
Hmm, I feel like this explanation applies to consistency checks in general, not really this one in specific.

I think it's worth explaining how it's used for checks that are weirdly specific like `monotonic_decrease_check` that's really just used for one thing, and is unlikely to have other applications, but less so for these checks that are very general and actually are used for many things. If someone wants to find out what we use this check for, they are probably better served by looking at our pipeline definitions.

Reply:
I've removed the `higher resolution` wording